app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -46,22 +47,33 @@
 
     private SearchView searchView;
 
+    private TextView noLocationSessions;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noLocationSessions' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594499/issues/source?bid=3233850&fileBranchId=3233896#l50)